docs/LEARNING.md

```diff
@@ -1,2 +1,10 @@
 # Learning
 
+ARM64 assembly is a simple assembly language. 
+You might be able to start off with [a reference card](https://courses.cs.washington.edu/courses/cse469/19wi/arm64.pdf).
```

Comment:
Reference links, as per [the Exercism markdown guide][1].

[1]: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md